tasks/marketplacesvm-push-disk-images/tests/test-marketplacesvm-push-disk-images.yaml

```diff
@@ -0,0 +1,190 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-marketplacesvm-push-disk-images
+spec:
+  description: |
+    Run the marketplacesvm-push-disk-images task with valid data to ensure it succeeds
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      taskSpec:
+        steps:
+          - name: setup
+            image: quay.io/konflux-ci/release-service-utils:7f7a156a835c773bbcd7e5d7e44df2f573db14f2
+            script: |
+              #!/usr/bin/env bash
+              set -eux
+
+              cat > "$(workspaces.data.path)/snapshot_spec.json" << EOF
+              {
+                "application": "amd-bootc-1-3-raw-disk-image",
+                "artifacts": {},
+                "components": [
+                  {
+                    "containerImage": "quay.io/workload/tenant/test-product/amd-bootc-1-3-raw-disk-image@sha256:123456",
+                    "name": "amd-bootc-1-3-raw-disk-image",
+                    "source": {
+                      "git": {
+                        "revision": "1abbfcdbc1c5e8b7aba07673297237ed192b50e2",
+                        "url": "https://gitlab.com/konflux/test-product/disk-images/amd-bootc"
+                      }
+                    }
```

Comment:
missing comma here I think? A comma is missing somewhere... that is why it is saying ` parse error: Expected separator between values at line 14, column 14`

Reply:
Thank! Fixed